android/uhabits-android/src/main/res/values/styles.xml

```diff
@@ -135,8 +136,17 @@
         <item name="headerBackgroundColor">@color/black</item>
         <item name="highlightedBackgroundColor">@color/black</item>
 
+        <item name="highContrastTextColor">@color/grey_200</item>
+        <item name="mediumContrastTextColor">@color/grey_700</item>
         <item name="lowContrastTextColor">@color/grey_900</item>
+
         <item name="highContrastReverseTextColor">@color/black</item>
+        <item name="mediumContrastReverseTextColor">@color/grey_900</item>
+        <item name="lowContrastReverseTextColor">@color/grey_700</item>
+
+        <item name="android:textColor">@color/grey_200</item>
+        <item name="textColorAlertDialogListItem">@color/grey_100</item>
```

Comment:
What problem are you trying to solve with these changes? Please include screenshots.

Reply:
In dark theme settings, for instance when we want to change the first day of week, a dialog opens up. The font color within this dialog is dark and hard to see. We suggest a lighter font color for this part of the app.
<img src="https://user-images.githubusercontent.com/71817053/100499307-c36d1d80-3168-11eb-9e52-f749d5c62521.jpg" width = "200px" height="356px"> <img src="https://user-images.githubusercontent.com/71817053/100499418-cd435080-3169-11eb-822f-f6c4dda40c14.jpg" width = "200px" height="356px">